pyfuntofem/model/scenario.py

```diff
@@ -72,12 +74,17 @@ def __init__(
             the total number of fun3d time steps to run for the scenario
         preconditioner_steps: int
             the number of fun3d iterations ran before coupled iterations for preconditioning
+        adjoint_steps: int
+            optional number of adjoint steps when using FUN3D analysis, can have different
+            number of forward and adjoint steps in steady-state
         T_ref: double
             Structural reference temperature (i.e., unperturbed temperature of structure) in Kelvin.
         T_inf: double
             Fluid freestream reference temperature in Kelvin.
         tacs_integration_settings: :class:`~interface.TacsUnsteadyInterface`
             Optional TacsIntegrator settings for the unsteady interface (required for unsteady)
+        fun3d_project_name: filename
```

Comment:
This should only matter for aerodynamic shape optimizations, correct? Otherwise, FUNtoFEM isn't touching the .ugrid file.